Bottom: cfec50c19b11f1486a7b674fa2a419f3056b3d43
Top:    a3bd52441d4287dd0d35f707b264b5a71fb8e0c5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-07 17:30:07 +0000

Refresh of remove-my-memory-allocation-ro

---

diff --git a/src/netlist.c b/src/netlist.c
index 6c5ca26..4a26bc9 100644
--- a/src/netlist.c
+++ b/src/netlist.c
@@ -243,9 +243,7 @@ static void
 netlist_style (LibraryMenuType *net, const char *style)
 {
   free (net->Style);
-  net->Style = NULL;
-  if (style)
-    net->Style = strdup ((char *)style);
+  net->Style = STRDUP ((char *)style);
 }
 
 /* The primary purpose of this action is to rebuild a netlist from a
